Use explicit 2.0.2 tag for eleveldb

jadeallenx opened the pull request:
Instead of pulling the branch, we now pull an explicit tag for eleveldb.

Commit:
Use explicit 2.0.2 tag for eleveldb

cmeiklejohn commented:
I'm confused.  The whole point of the `develop` branch is that all branches would be on develop.  Why is this required?

andrewjstone committed:
Fix 'make test' and minimize work done in all

This matches the rest of our repos

cmeiklejohn committed:
Ignore test if EQC is not available.

This is required for the builders, where there is no EQC when attempting
to validate the build.

cmeiklejohn committed:
Use custom script for tests.

Use the custom script for running the tests; override the tools.mk
target.

cmeiklejohn committed:
Test target needs testdeps.

andrewjstone commented:
:+1: cd98cd0

cmeiklejohn commented:
@borshop merge
